feature1/one-more-file.txt

```diff
@@ -0,0 +1,3 @@
+this file was also aded in the feature-1 branch.
```

Comment:
It's really good that you have decided to add this line of code

Reply:
Thanks for this comment, El Mehdi